README.md

```diff
@@ -299,7 +297,7 @@ docker ps
 docker exec -it djangoapp sh
 
 # run manage.py inside container shell
-SECRET_KEY=whatever python manage.py migrate --settings=config.settings.local-container
+SECRET_KEY=whatever python manage.py migrate --settings=config.settings.container
 ```
 
 ### Deploying a new version
```

Comment:
I think one important aspect is still missing: Which files should ALWAYS be copied (and redacted) from the repo to the server, when deploying. I guess, something like:
```bash
scp .env.<production|test> monitoring@monitoring.localzero.net:/data/<production|test>/.env
```
will be the minimum. Then also generating the secret key and replacing it in the file.

Reply:
Yes, that's missing becuase I haven't actually tried that part yet. I'll add it once I know what needs to be done.